src/ruby/test/mml_test.rb

```diff
@@ -14,14 +14,14 @@ def basic_case
 
   description "A flat"
   def a_flat_case
-    duration = MML.new.compile("AA-") { |p, d| @data << [p, d] }
+    duration = MML.new.compile("AA-") { |p, d| @data << [p.to_i, d] }
```

Comment:
Is there any better way?

Reply:
I think you should edit the data of the next line `[[440, 500],[415,500]],` to an actual float value

Reply:
OK, I did.

Although it failed when I wrote %.3f values which is along the result outputs.